src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -232,21 +296,31 @@ export default class AuthorizationService {
       tenant, userToken, Entity.USER, Action.UPDATE, authorizationFilter, { UserID: user.id }, user);
     user.canDelete = await AuthorizationService.canPerformAuthorizationAction(
       tenant, userToken, Entity.USER, Action.DELETE, authorizationFilter, { UserID: user.id }, user);
+    user.canAssignSites = await AuthorizationService.canPerformAuthorizationAction(
```

Comment:
In addUsersAuthorizations (line 278) you are missing some auth flags to drive the GUI: canListSites, canListTags etc.
These would show/hide the filters when looking at the user list

Reply:
✅ modified in next commit